nengo_gui/ipython.py

```diff
@@ -33,7 +33,7 @@ class ConfigReuseWarning(UserWarning):
     pass
 
 
-class IPythonViz(object):
+class InlineGUI(object):
```

Comment:
I am very much on-board with renaming this, but we'll definitely want a deprecation system here, for backwards compatibility...

Reply:
Added deprecation warnings.